#17 Issue17 main window

Tymoteusz22 opened the pull request:
Basic refactor; Hashowanie haseł; 

TODO:
ekrany: Game Setting Window; Game Window;

Tymoteusz22 committed:
Duzy update do wszystkich gui

Tymoteusz22 committed:
Cos dziala

Tymoteusz22 committed:
Cos dziala

Tymoteusz22 committed:
#17 v1 ekranow juz blisko

Sirmarianus reviewed (commented):
pomyłka

Sirmarianus reviewed (approved):
sha512 totalny gicior

Tymoteusz22 committed:
chyba naprawione błędy logowania

Tymoteusz22 committed:
# 17 jednak nie tak blisko

Tymoteusz22 committed:
# 17 opisy do odkomentowania TODO (do oficjalnego dzialania)
Podsumowujac zostalo: mini-ekran logowania, okno_z_gra (gameWindow)

Tymoteusz22 committed:
#17 importy wzgledne zamiast bezwzglednych

Tymoteusz22 committed:
#17 powoli zaczynamy gameWindow
-Modyfikacje nazw plików na angielskie (resources)
-Początek wyświetlania Płytek
-Liczne błędy i niepotrzebne linie/próby dodania czegoś w gratisie, bo czemu nie ;)
